### PR TITLE
Apply MQE on K8s layer UI-templates

### PR DESCRIPTION
**Pull request by smartboy37597**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes [#10914](https://github.com/apache/skywalking/issues/10914)
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

Here are some UI and e2e test screenshots.
ui-5
ui-3
k8su1-2
k8s-ui
k8se2e-success

### CODE REVIEW

File: skywalking-ui
Comment (wankai123) [inline skywalking-ui:1]: It seems an old version. A mistake sync?

Comment (wu-sheng) [reply skywalking-ui:1]: I think it should be a mistake. @smartboy37597 Please revert it.

Comment (smartboy37597) [reply skywalking-ui:1]: It should be a sync error, I will try to solve it.

**Comment by wu-sheng** [comment]
ebpf cases have errors. Something is not matched. cc @mrproliu

File: test/e2e-v2/cases/profiling/ebpf/network/expected/metrics-has-client-value-label.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# limitations under the License.
+
+
+type: TIME_SERIES_VALUES
+results:
+  {{- contains .results }}
+  - metric:
+      labels:
+        - key: _
+          value: "false-http-172.18.0.6:80:50"
```
Comment (mrproliu) [inline test/e2e-v2/cases/profiling/ebpf/network/expected/metrics-has-client-value-label.yml:23]: Does the value of the label look like a dynamic IP?

Comment (wu-sheng) [reply test/e2e-v2/cases/profiling/ebpf/network/expected/metrics-has-client-value-label.yml:23]: I feel different KinD would provide different IP pools. So, don't use the specific addresses.

Comment (smartboy37597) [reply test/e2e-v2/cases/profiling/ebpf/network/expected/metrics-has-client-value-label.yml:23]: `process_relation_client_write_rtt_time_percentile` and `process_relation_client_write_exe_time_percentile`
`process_relation_server_write_rtt_time_percentile` and `process_relation_server_write_exe_time_percentile`
Can the metrics return the same value as `process_relation_http1_request_package_size_percentile` and `process_relation_http1_response_package_size_percentile` is consistent, and I think the label value of `http1` is better.

File: skywalking-ui
Comment (smartboy37597) [reply skywalking-ui:1]: ui is synchronized

File: test/e2e-v2/cases/profiling/ebpf/network/expected/metrics-has-client-value-label.yml (hunk 1 above)
Comment (wu-sheng) [reply test/e2e-v2/cases/profiling/ebpf/network/expected/metrics-has-client-value-label.yml:23]: How TCP could be as same as HTTP? 
This PR should keep a metric to MQE, let's don't introduce more changes in here.

Comment (mrproliu) [reply test/e2e-v2/cases/profiling/ebpf/network/expected/metrics-has-client-value-label.yml:23]: I think it is enough for you to verify that the value of the label is not empty.

Comment (smartboy37597) [reply test/e2e-v2/cases/profiling/ebpf/network/expected/metrics-has-client-value-label.yml]: I've fixed it and the e2e test are passing.
image

**Comment by wu-sheng** [comment]
I will take care of `Dead Link Checker ` through another PR. It seems some links had been removed from Envoy doc site.

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s/k8s-cluster.json [hunk 2]
```diff
@@ -1,19 +1,3 @@
-/**
```
Comment (mrproliu) [inline oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s/k8s-cluster.json:1]: @kezhenxu94 Should we keep the license header in the UI template files?

Comment (wankai123) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s/k8s-cluster.json:1]: I think we no longer need these headers: https://github.com/apache/skywalking/pull/11184#discussion_r1285683989

Comment (wu-sheng) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s/k8s-cluster.json:1]: @smartboy37597 Could you follow up on another PR to remove other existing headers to keep them consistent?

**Review by wankai123 — APPROVED** [review]
LGTM

**Review by mrproliu — APPROVED** [review]

**Comment by wu-sheng** [comment]
@smartboy37597 We only have OpenFunction and topology related settings pending on the immigration to MQE.
Take your time to finish the final step.

File: oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s/k8s-cluster.json (hunk 2 above)
Comment (smartboy37597) [reply oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s/k8s-cluster.json:1]: > @smartboy37597 Could you follow up on another PR to remove other existing headers to keep them consistent?

Okey, I'm going to delete all UI headers in a new PR.

**Comment by smartboy37597** [comment]
> @smartboy37597 We only have OpenFunction and topology related settings pending on the immigration to MQE. Take your time to finish the final step.

Okey, i got it.

**Pull request merged**